src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const activeImg = document.querySelector('.gallery__large-img');
+const thumb = document.querySelector('.gallery__list');
+
+thumb.addEventListener('click', (clickevent) => {
+  const link = clickevent.target.closest('.list-item__link');
+  const img = clickevent.target.closest('.gallery__thumb');
+
+  clickevent.preventDefault();
+
+  if (!link || !img) {
+    return;
+  }
+
+  const imgSrc = link.getAttribute('href');
+
+  activeImg.removeAttribute('src');
```

Comment:
The `removeAttribute('src')` call is unnecessary here. Setting a new `src` attribute will automatically replace the old one. Consider removing this line to simplify the code.